lib/vagrant-service-manager/binary_handlers/cdk_openshift_binary_handler.rb

```diff
@@ -1,7 +1,13 @@
 module VagrantPlugins
   module ServiceManager
-    class CDKOpenshiftBinaryHandler < CDKBinaryHandler
+    # Currently client binary installation of openshift for CDK is same as ADB
+    class CDKOpenshiftBinaryHandler < ADBOpenshiftBinaryHandler
+      # Default to latest stable origin oc version for CDK as it is different than
+      # OSE oc version running inside CDK development environment
+      LATEST_OC_VERSION = '1.2.1'.freeze
+
       def initialize(machine, env, options)
+        options['--cli-version'] = LATEST_OC_VERSION unless options['--cli-version']
```

Comment:
@hferentschik  `--cli-version` option is still supported for CDK client binary installation. Only in case of openshift, since there was no way to get the origin compatible version from CDK, I have pinned it to latest stable release of oc i.e `1.2.1`.